src/AutorestSwift/View Models/ServiceClientFileViewModel.swift

```diff
@@ -35,18 +35,46 @@ struct ServiceClientFileViewModel {
     let apiVersionName: String
     let protocols: String
     let paging: Language.PagingNames?
+    let globalParameters: [ParameterViewModel]
+    // A dictionary of all the named operation group. Key is the group name , Value is the operation group view model.
+    let namedOperationGroups: [String: OperationGroupViewModel]
+    // A key,Value pairs of all the named operation group for stencil template engine
+    let namedOperationGroupProperties: [KeyValueViewModel]
```

Comment:
Wouldn't these live inside the `OperationGroupViewModel`? Why are they here?

Reply:
This is for creating properties for each named operation group in ServiceClient.